modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -175,29 +312,29 @@ For example:
 `$ curl -H "Authorization: Bearer [YOUR_ACCESS_TOKEN]" https://anypoint.mulesoft.com/accounts/api/me`
```

Comment:
Look at all the URLs and escape the ones you don't want to be hyperlinks.

Reply:
These are intentionally not escaped.